modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -185,127 +177,115 @@ def decision_reviews_security_schemes
+        sc_response_schemas,
+        sc_alt_signer_schemas,
+        contestable_issues_schema.slice(*%i[contestableIssue]),
+        generic_schemas.except(*%i[errorWithTitleAndDetail timeStamp uuid X-Consumer-ID X-Consumer-Username X-VA-NonVeteranClaimant-SSN X-VA-NonVeteranClaimant-Birth-Date]),
+        shared_schemas.slice(*%W[address phone timezone #{nbs_key}])
       )
-      a << shared_schemas.slice(*%W[address phone timezone #{nbs_key}])
     when 'appealable_issues'
-      a << appealable_issues_schema('#/components/schemas')
-      a << generic_schemas('#/components/schemas').slice(*%i[errorModel X-VA-SSN X-VA-File-Number X-VA-ICN])
-      a << shared_schemas.slice(*%W[#{nbs_key}])
+      merge_schemas(
+        appealable_issues_schema,
+        generic_schemas.slice(*%i[errorModel X-VA-SSN X-VA-File-Number X-VA-ICN]),
+        shared_schemas.slice(*%W[#{nbs_key}])
+      )
     when 'legacy_appeals'
-      a << legacy_appeals_schema('#/components/schemas')
-      a << generic_schemas('#/components/schemas').slice(*%i[errorModel X-VA-SSN X-VA-File-Number X-VA-ICN])
-      a << shared_schemas.slice(*%W[#{nbs_key}])
+      merge_schemas(
+        legacy_appeals_schema,
+        generic_schemas.slice(*%i[errorModel X-VA-SSN X-VA-File-Number X-VA-ICN]),
+        shared_schemas.slice(*%W[#{nbs_key}])
+      )
     when 'appeals_status'
-      a << appeals_status_response_schemas
-      a << generic_schemas('#/components/schemas').slice(*%i[errorModel X-VA-SSN])
+      merge_schemas(
+        appeals_status_response_schemas,
+        generic_schemas.slice(*%i[errorModel X-VA-SSN])
+      )
     when 'decision_reviews'
-      a << hlr_v2_create_schemas
-      a << hlr_v2_response_schemas('#/components/schemas')
-      a << nod_v2_create_schemas
-      a << nod_v2_response_schemas('#/components/schemas')
-      a << sc_create_schemas
-      a << sc_response_schemas('#/components/schemas')
-      a << sc_alternate_signer_schemas('#/components/schemas')
-      a << contestable_issues_schema('#/components/schemas')
-      a << legacy_appeals_schema('#/components/schemas')
-      a << generic_schemas('#/components/schemas')
-      tmp = shared_schemas.tap { |h| h['nonBlankString'] = h.delete('non_blank_string') }
-      a << tmp
+      merge_schemas(
+        decision_reviews_hlr_create_schemas,
+        decision_reviews_hlr_response_schemas,
+        decision_reviews_nod_create_schemas,
+        decision_reviews_nod_response_schemas,
+        decision_reviews_sc_create_schemas,
+        decision_reviews_sc_response_schemas,
+        decision_reviews_sc_alt_signer_schemas,
+        contestable_issues_schema,
+        legacy_appeals_schema,
+        generic_schemas(nbs_key:),
+        shared_schemas(nbs_key:)
+      )
     else
       raise "Don't know how to build schemas for '#{api_name}'"
     end
-
-    a.reduce(&:merge).sort_by { |k, _| k.to_s.downcase }.to_h
   end
-  # rubocop:enable Metrics/AbcSize
 
-  def generic_schemas(ref_root)
```

Comment:
`ref_root` was the same everywhere, so I changed references to it into a string literal to keep things simple